modules/apps/object/object-test/src/testFunctional/paths/ObjectRelationships.path

```diff
@@ -26,8 +26,8 @@
 		<td></td>
 	</tr>
 	<tr>
-		<td>OPTIONS_TAB_RELATIONSHIPS_TYPE_AND_HIERARCHY</td>
-		<td>//div[contains(@class,'dnd-tr') and contains(.,'${key_relationType}')and contains(.,'${key_relationHierarchy}')]</td>
+		<td>OPTIONS_TAB_RELATIONSHIPS_HIERARCHY_AND_LABEL</td>
+		<td>//div[contains(@class, 'dnd-td') and contains(., '${key_relationHierarchy}')]//preceding-sibling::div//*[contains(text(), '${key_labelName}')]</td>
```

Comment:
You're basically getting rid of a path that being used in ObjectRelationships.testcase and replacing it with your own. This is going to break several tests that rely on the old path.

Reply:
This path allows you to filter a relationship by Label and Hierarchy. In cases like the MtoMany relationship, the old path wouldn't work because it only filters by the Label, in which case the Labels are the same.

But since this is the only case, I'm going to undo the changes. Thanks!